scripts/docker-build-all.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)
+
+OPENMETHANE_PATH=${OPENMETHANE_PATH:-"$SCRIPT_DIR/.."}
+OPENMETHANE_PRIOR_PATH=${OPENMETHANE_PRIOR_PATH:-"$OPENMETHANE_PATH/../openmethane-prior"}
+SETUP_WRF_PATH=${SETUP_WRF_PATH:-"$OPENMETHANE_PATH/../setup-wrf"}
```

Comment:
This script assumes that `openmethane-prior` and `setup-wrf` are checked out in sibling folders next to `openmethane`, with the same name as the (current) git repo. If your setup looks a bit different, you can manually supply the paths via:

```shell
OPENMETHANE_PRIOR_PATH=/path/to/prior SETUP_WRF_PATH=/path/to/setup_wrf scripts/docker-e2e-daily.sh
```